apps/frontend/src/renderer/components/linear/components/LinearTicketDetail.tsx

```
@@ -0,0 +1,235 @@
+        ) : (
+          <div className="flex items-center gap-1 px-2 py-1 bg-gray-50 dark:bg-gray-800 rounded-full" role="status" aria-live="polite">
+            <Clock className="w-4 h-4 text-gray-400" aria-hidden="true" />
+            <span className="text-xs text-gray-500">
+              {t('linear:notValidated')}
+            </span>
+          </div>
+        )}
+      </div>
+
+      {/* Metadata Badges */}
+      <div className="flex flex-wrap gap-2" role="list" aria-label="Ticket metadata">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)